packages/test/CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 2.0.0-k.2 (2021-01-06)
+
+* enhance: Remove FixtureManager in favor of MockResolver (#447) ([4aa1617](https://github.com/coinbase/rest-hooks/commit/4aa1617)), closes [#447](https://github.com/coinbase/rest-hooks/issues/447)
+* feat: Add <MockResolver /> (#446) ([614576a](https://github.com/coinbase/rest-hooks/commit/614576a)), closes [#446](https://github.com/coinbase/rest-hooks/issues/446)
```

Comment:
would it be possible to only include line 9 `feat: Add <MockResolver />` (as the other commits are gone in #447)